pkg/operator/controller/names.go

```diff
@@ -22,6 +22,10 @@ const (
 	// GlobalUserSpecifiedConfigNamespace is the namespace for configuring OpenShift.
 	GlobalUserSpecifiedConfigNamespace = "openshift-config"
 
+	// IngressOperatorOwnedAnnotation is the key for the annotation that
+	// indicates that the ingress operator owns the annotated resource.
+	IngressOperatorOwnedAnnotation = "ingress.operator.openshift.io/owned"
```

Comment:
Why is this needed?

Reply:
There was a suggestion on a recent working-group call that we do something to help us distinguish in the future whether the subscription was created by the operator or by the cluster-admin (or some other entity).  This might turn out to be critical to the OSSM upgrade logic for OCP 4.19→4.20.  Anyway, it doesn't hurt to have this annotation.

Reply:
nit: It would be nice to ensure that future developers don't use this annotation on a wider basis because they're unsure why it's here.  Could you add a comment that it is really only supposed to be used by subscriptions?

Reply:
Sorry, I missed this comment before.  https://github.com/openshift/cluster-ingress-operator/pull/1152/commits/5c76a8511cee19aeec606b2dffd70d97cf595828 expands the comment to say that the annotation is currently only intended to be applied to newly created subscriptions.